src/middleware/authentication.ts

```diff
@@ -11,14 +11,15 @@ export const isAuthenticated = (
   passport.authenticate(
     'jwt',
     { session: false },
-   (err: Error, user: UserAttributes) => {
+    (err: Error, user: UserAttributes) => {
       if (!user) {
         return sendResponse(res, 403, null, 'Please login...');
       }
       const currUser = {
         id: user.id,
         name: user.name,
         email: user.email,
+        password: user.password,
```

Comment:
we don't need password please remove it  you have id of user please remove password